### PR TITLE
update exceptions logic

### PR DESCRIPTION
**Pull request by podolinek**
updated exceptions nesting - one base exception for better catching potential problems with library

### CODE REVIEW

File: src/SmsManager/Exceptions/SmsManagerException.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+/**
+ * Part of jakubenglicky/sms-manager
+ * @author Jakub Englický
+ */
+
+namespace jakubenglicky\SmsManager\Exceptions;
+
+class SmsManagerException extends SmsManagerException
+{
+}
```
Comment (jakubenglicky) [inline src/SmsManager/Exceptions/SmsManagerException.php]: new line at end of file

**Review by jakubenglicky — COMMENTED** [review]
Thank you for your contribution. Please fix end of file, after fixup PR will be merged.

**Review by jakubenglicky — APPROVED** [review]
Big thx, please squash

**Comment by podolinek** [comment]
Any action from me required? dunno what is sqash... :]

**Comment by jakubenglicky** [comment]
Released – https://github.com/jakubenglicky/sms-manager/releases/tag/v2.1.0
Great article about working with git (fixup, squash) – https://filip-prochazka.com/blog/git-fixup

**Pull request merged**